src/convertToObject.js

```diff
@@ -5,8 +5,22 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleArr = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style !== '');
```

Comment:
Consider improving the indentation of chained methods to match the good example provided in the checklist.